src/app/service/mockdata.ts

```diff
@@ -41,6 +41,9 @@ const accountingFigures = Object.entries(rawAccountingFigures).reduce(
         return allFigures;
     }, {} as AccountingFigures);
 
+const companyValue = 122000;
```

Comment:
Ist es nicht so, dass durch den stochastischen Ansatz bei allen drei Verfahren unterschiedliche Unternehmenswerte "berechnet" werden? Will man das in de Mockdaten aufzeigen?

Reply:
Oh, ich dachte, man kommt immer beim selben raus. Ka, wie's wirklich ist...